cmd/bundle/plan.go

```diff
@@ -79,14 +80,60 @@ func newPlanCommand() *cobra.Command {
 			return root.ErrAlreadyPrinted
 		}
 
+		// Count actions by type and collect formatted actions
+		createCount := 0
+		updateCount := 0
+		deleteCount := 0
+		changed := make(map[string]bool)
+
+		for _, change := range plan.GetActions() {
+			changed[change.Group+"."+change.Key] = true
+			switch change.ActionType.String() {
+			case "create":
+				createCount++
+			case "update", "update_with_id":
```

Comment:
if you rebase, you can use StringShort() and that will be "update" for both of these.